src/vs/vscode.d.ts

```diff
@@ -1949,6 +1949,48 @@ declare module 'vscode' {
 	}
 
 	/**
+	 * Represents information about a programming construct that can be used to
+	 * identify and locate the construct's symbol. The identification does not have
+	 * to be unique, but it should be as unique as possible. It is up to the
+	 * language server to define the schema of this object.
+	 *
+	 * In contrast to `SymbolInformation`, `SymbolDescriptor` includes more concrete,
+	 * language-specific, metadata about the symbol.
+	 */
```

Comment:
Yeah, comments should be persisted by tsc. Which is why I wondered why the comments are different - maybe you forgot to compile in the end?